examples/placeholder/linux/AppOptions.h

```diff
@@ -0,0 +1,34 @@
+/*
+ *
+ *    Copyright (c) 2022 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include "AppMain.h"
+
+#include <app/tests/suites/credentials/TestHarnessDACProvider.h>
+
+class AppOptions
+{
+public:
+    static chip::ArgParser::OptionSet * GetOptions();
+    static chip::Credentials::DeviceAttestationCredentialsProvider * GetDACProvider();
+
+private:
+    static bool HandleOptions(const char * program, chip::ArgParser::OptionSet * options, int identifier, const char * name,
+                              const char * value);
+};
```

Comment:
Why is the main Options.h not good enough for this?

Reply:
Mostly because someone if some apps does not use the test harness version of the Dax provider if looks odd to have the "--dac_provider" option. But we can move it later if you want.